### PR TITLE
Erik itemslayout

### PR DESCRIPTION
**Pull request by egall18**
Making changes to how to pass in data to the html for items. I am using a arraylist<map<string,string>> to pass in the information and accessing the values with .get() method. 
Screen Shot 2021-10-14 at 10 25 42 AM

### CODE REVIEW

**Comment by egall18** [comment]
Yes because all we are doing here is removing the quotes from the values. If we were to append a new item then that would be stored as a string as well.

**Comment by GuillermoFloresV** [comment]
okay sounds good, marked this as still in a draft for now, ping me if any issues come up/need help

File: src/main/java/csumb/edu/project2/controller/WebController.java [hunk 1]
```diff
@@ -72,7 +75,7 @@ public String index(Model model, HttpServletRequest request) throws IOException
 
     @RequestMapping("/myitems")
     public String items(Model model, HttpServletRequest request) throws IOException {
-        String apiURL = "https://radiant-cliffs-80770.herokuapp.com/items";
+        String apiURL = "http://localhost:8080/items";
```
Comment (roberthuskins) [inline src/main/java/csumb/edu/project2/controller/WebController.java:78]: String apiURL = urlFetcher.getUrl() + "/items";

Comment (GuillermoFloresV) [reply src/main/java/csumb/edu/project2/controller/WebController.java:78]: I second this

Comment (egall18) [reply src/main/java/csumb/edu/project2/controller/WebController.java]: fixed

**Pull request merged**